### PR TITLE
add download subcommand for xref.

### PR DESCRIPTION
**Pull request by vwxyzh**
#282

### CODE REVIEW

File: src/docfx/Models/DownloadCommandOptions.cs [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+    using CommandLine;
+
+    [OptionUsage("download [<output xref archive>]")]
+    internal class DownloadCommandOptions : ICanPrintHelpMessage
+    {
+        [ValueOption(0)]
+        public string ArchiveFile { get; set; }
+
+        [Option('x', "xref", HelpText = "Specifies the urls of xrefmap used by content files.")]
+        public string Uri { get; set; }
```
Comment (superyyrrzz) [inline src/docfx/Models/DownloadCommandOptions.cs]: What about: `"Specify the url of xrefmap used by content files.`?

**Comment by qinezh** [comment]
:+1:

File: src/docfx/Models/DownloadCommandOptions.cs (hunk 1 above)
Comment (vwxyzh) [inline src/docfx/Models/DownloadCommandOptions.cs]: forgot to modify it ...

File: src/docfx/Models/DownloadCommandOptions.cs [hunk 2]
```diff
@@ -11,7 +11,7 @@ internal class DownloadCommandOptions : ICanPrintHelpMessage
         [ValueOption(0)]
         public string ArchiveFile { get; set; }
 
-        [Option('x', "xref", HelpText = "Specifies the urls of xrefmap.")]
+        [Option('x', "xref", HelpText = "Specifies the url of xrefmap.")]
```
Comment (superyyrrzz) [inline src/docfx/Models/DownloadCommandOptions.cs]: Specifies => Specify?

**Comment by superyyrrzz** [comment]
:+1:

**Pull request merged**